### PR TITLE
bumping fluent bit version to 3.0.4

### PR DESCRIPTION
**Pull request by nr-dsharma**
This will fix vulnerabilities https://nvd.nist.gov/vuln/detail/CVE-2024-4323

### CODE REVIEW

File: version.go [hunk 1]
```diff
@@ -1,3 +1,3 @@
 package main
 
-const VERSION = "1.19.2"
```
Comment (luckslovez) [inline version.go:3]: I'd say this deserves a major version bump

Comment (nr-dsharma) [reply version.go:3]: should be have 2.0.0 or 1.20.0 ?

Comment (luckslovez) [reply version.go:3]: Just a refresher! [MAJOR.MINOR.PATCH](https://semver.org/)

**Review by luckslovez — APPROVED** [review]
Nice!

**Pull request merged**